java/fury-benchmark/src/main/java/io/fury/benchmark/data/Data.java

```diff
@@ -1,21 +1,18 @@
-/* Copyright (c) 2008-2023, Nathan Sweet
- * All rights reserved.
+/*
```

Comment:
The data is coming from kryo benchmark, maybe we should reserve its header.

Reply:
Thanks for your review! These files are not included in the LICENSE file. I'll revert them at once after you finish the check (please let me know when it finished). This also indicates that we can hold single file copied in each file, while significant module copied or dependencies in the NOTICE file.